{"name": "test_alert2_failure", "status": "passed", "steps": [{"name": "Нажатие кнопки для вызова Alert 2", "status": "passed", "start": 1760098296266, "stop": 1760098296379}, {"name": "Отменить Cancel у Alert 2", "status": "passed", "start": 1760098296380, "stop": 1760098296385}, {"name": "Подтверждаем, что после нажатия Отмены у нас выпало нужное предложение", "status": "passed", "steps": [{"name": "Получить текст после нажатия Alert 2", "status": "passed", "start": 1760098296418, "stop": 1760098296439}], "start": 1760098296385, "stop": 1760098296439}], "start": 1760098296224, "stop": 1760098296439, "uuid": "19a45a92-5d77-431e-9fde-26560a9a5fa6", "historyId": "ff70814ac359c54dad1faef5f68cde8f", "testCaseId": "ff70814ac359c54dad1faef5f68cde8f", "fullName": "test_alerts.TestAlertsPage#test_alert2_failure", "labels": [{"name": "feature", "value": "Alerts"}, {"name": "story", "value": "Тест второго Alert (Негативный - нажатие отмены)"}, {"name": "suite", "value": "test_alerts"}, {"name": "subSuite", "value": "TestAlertsPage"}, {"name": "host", "value": "DESKTOP-T28TT8C"}, {"name": "thread", "value": "10144-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alerts"}], "titlePath": ["test_alerts.py", "TestAlertsPage"]}